### PR TITLE
Fix host_system_name for linux

### PR DESCRIPTION
**Pull request by tsawada**
This looks like some copy-paste leftover but I'm not sure how bazel uses this info.

### CODE REVIEW

**Review by d-asnaghi — CHANGES_REQUESTED** [review]
Amazing work @tsawada, thanks for this addition. These metadata are not actively used by Bazel as far as I know, but this is much more consistent.

May I ask you to just go all the way and also change the 
`toolchain_identifier` for linux_aarc64 while you're at it?

Thanks!

File: toolchain/arm-none-eabi/linux_aarch64/BUILD [hunk 1]
```diff
@@ -65,7 +65,7 @@ cc_arm_none_eabi_config(
     name = "config",
     gcc_repo = compiler,
     gcc_version = "9.2.1",
-    host_system_name = "x86-darwin",
+    host_system_name = "linux_aarch64",
```
Comment (d-asnaghi) [inline toolchain/arm-none-eabi/linux_aarch64/BUILD:68]: We should also change the next line if possible

Comment (tsawada) [reply toolchain/arm-none-eabi/linux_aarch64/BUILD:68]: didn't notice :) fixed.

**Comment by d-asnaghi** [comment]
Thank you @tsawada

**Pull request merged**